src/content/docs/build/env-variables/store_environment-variables.mdx

```diff
@@ -0,0 +1,36 @@
+---
+page_id: 2d39a390-baf8-4f49-a18c-950d3f248c65
+title: Store environment variables in Kinde
+sidebar:
+  order: 1
+relatedArticles:
+  - 82646df5-5e4c-4b09-babf-2f77c36bd73b
+  - d95a8edd-f61a-4095-a80e-ffd67f90a438
+---
+
+
+Environment variables are key-value pairs that enable access to applications across different environments and systems. 
+
+Now that you can bring your code into Kinde via workflows, it’s important to be able to reference  environment variables using a key, instead of hardcoding these values into your code.
+
+Kinde allows you to create and store sensitive (secret) and non-sensitive key-value pairs.
+
+## Store variables for each environment
+
+You need to set environment variables separately, in each of your Kinde environments. You can’t share keys across development, testing, staging, and production, etc.
+
+## Sensitive and non-sensitive environment variables
+
+The environment variables you store in Kinde can be sensitive or non-sensitive. 
+
+- **Sensitive environment variables** contain confidential data (e.g., passwords, API keys, credentials) that must be handled with care to avoid security risks and data breaches.
```

Comment:
When you mark an existing environment variable as sensitive, Kinde converts it to a non-readable format and the value can no longer be read by you or your team. It can only be read by Kinde workflows at runtime. The value will also be redacted in any logs.

Reply:
Added